server/tasks/github_webhook_tasks.py

```
@@ -0,0 +1,988 @@
+        raise ValueError("payload missing 'installation' object")
+
+    installation_id = installation.get("id")
+    if not isinstance(installation_id, int):
+        raise ValueError("payload missing 'installation.id' int")
+
+    account = installation.get("account") or {}
+    account_login = account.get("login") if isinstance(account, dict) else None
+    return installation_id, installation, account_login or ""
+
+
+def _handle_installation_event(
```

Ruff (0.15.12)

[warning] Too many statements (66 > 50)

(PLR0915)